docroot/themes/humsci/humsci_basic/src/scss/admin/_wysiwyg-editor.scss

```diff
@@ -98,3 +98,16 @@ drupal-media {
 .ck-editor .ck.ck-toolbar.ck-toolbar_grouping > .ck-toolbar__items {
   flex-wrap: wrap;
 }
+
+body.gin--edit-form .views-exposed-form .select-preact {
```

Comment:
Please move the code to `docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.css`. That's the place to add admin theme styles.